doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,295 @@
+# Environment Variables in Julia
+
+The environment variables that Julia uses generally start with `JULIA`. If
+[`Base.versioninfo()`](@ref) is called with `verbose` equal to `true`, then the
+output will list all environment variables for which `JULIA` appears in the name.
+
+
+## File locations
+
+### `JULIA_HOME`
+
+The absolute path of the directory containing the Julia
+executable, which sets the global variable [`Base.JULIA_HOME`](@ref). If
+`$JULIA_HOME` is not set, then `Base.JULIA_HOME` defaults to its compile-time
+value.
+
+The executable itself is one of
```

Comment:
Most of the docs leave a single blank line before and after triple backtick blocks. Makes it a bit easier to see where code blocks start and end if they don't specify a language.